lib/workers/repository/init/vulnerability.ts

```diff
@@ -125,11 +127,13 @@ export async function detectVulnerabilityAlerts(
       } else {
         logger.debug('Invalid firstPatchedVersion: ' + firstPatchedVersion);
       }
+      alertDetails.fileType = fileType;
     } catch (err) {
       logger.warn({ err }, 'Error parsing vulnerability alert');
     }
   }
   const alertPackageRules: PackageRule[] = [];
+  config.remediations = {} as any;
```

Comment:
```suggestion
  config.remediations = {} as never;
```

Reduce `any` usage. Is the cast necessary?